internal/tracing/interfaces.go

```diff
@@ -0,0 +1,11 @@
+package tracing
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+type TracingInterface interface {
+	Start(context.Context, string, ...trace.SpanStartOption) (context.Context, trace.Span)
+}
```

Comment:
if this is done becuase of the update of otel lib to `v1.21.0` then worth looking also at this implementation https://github.com/glauth/glauth/commit/1a3739610997b58100040d0c8a405596fccc8e23

now if this works i m happy with it, we can always review it down the line as i dunno the implications of not using the standard `Tracer`, at the same time if we only need `Start`, as we seem to be, all is good

Reply:
I added it to keep it consistent with the [monitoring](https://github.com/canonical/identity-platform-login-ui/blob/main/internal/monitoring/interfaces.go#L3) and [logging](https://github.com/canonical/identity-platform-login-ui/blob/main/internal/logging/interfaces.go#L3). No problem with removing it if you think it is out of place